run_sorting_hat_test_data.sh

```diff
@@ -10,22 +10,22 @@ SN_CONDA_PREFIX=${SN_CONDA_PREFIX:-$( pwd )/conda_installs}
 CONDA_PREFIX_LINE="--conda-prefix $SN_CONDA_PREFIX"
 export LOG_PATH=${LOG_PATH:-$( pwd )/sorting.log}
 USUAL_SM_ERR_OUT=${USUAL_SM_ERR_OUT:-$( pwd )/snakemake.log}
-pushd $1
 
+#create conda envs only
+for yaml_file in $(ls $( pwd )/envs); do
+  echo $yaml_file
+  snakemake --dry-run --use-conda --conda-create-envs-only $CONDA_PREFIX_LINE  \
```

Comment:
I think that --dry-run is redundant when using --conda-create-envs-only, on the successful run (see slack 02/09/2021 ~2:44) I didn't use --dry-run.

Reply:
done